packages/plugins/model-define/cycle-gan-model-define/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import { ModelDefineType, UniModel, ModelDefineArgsType, ImageSample, CocoDataset, ImageDataset } from '@pipcook/pipcook-core';
+import * as path from 'path';
+import * as fs from 'fs';
+
+const boa = require('@pipcook/boa');
+const { tuple } = boa.builtins();
+const sys = boa.import('sys');
+sys.path.insert(0, path.join(__dirname, '..'));
+const tf = boa.import('tensorflow');
+const { CycleGAN } = boa.import('CycleGAN.models');
+const np = boa.import('numpy');
+const cv2 = boa.import('cv2');
+const base64 = boa.import('base64');
+
+type PredictType = 'a2b' | 'b2a';
+
+let opt = {
+  // data
+  DATA_ROOT: '',         // path to images (should have subfolders 'train', 'val', etc)
+  shapeA: tuple([ 128, 128, 3 ]),
```

Comment:
These shapes should be dynamically read from dataset given by data-process / data-access.

Reply:
The model will resize the input images to the same shape size, and it's a model property.